6.0

lbrant1 opened the pull request.

ruttenb committed:
Bumped seeker version to 4.3.0

ruttenb committed:
Changes to export_value and columnn render:

1. Call value_format for value and highlight
2. Call value_format after the AttrList highlight format
3. In export_value, format AttrList before we call value_format

lefcourn committed:
Merge pull request #80 from ruttenb/updates_for_value_format

Changes to export_value and columnn render:

ruttenb committed:
Bump seeker versino to 4.3.1

ruttenb committed:
Check if highlight exists before calling value_format

ruttenb committed:
Bumped Seeker version to 4.3.2

ruttenb committed:
Updated build_filter_dict so it doesn't break if the default
'apply_aggregations' isn't called

ruttenb committed:
Added the inline comment back

lefcourn committed:
Merge pull request #81 from ruttenb/updates_to_build_filter_dict

Updated build_filter_dict so it doesn't break if the default

ruttenb committed:
Bumped seeker version to 4.3.3

smithlc committed:
Add field definitions

Commit:
Remove sort in normalize_querystring on values, fix bug for range filter

Sorting a range filter would remove the filter if only the less than is
set, and could potentially reverse the filters.

Commit:
Merge branch '6.0' of https://github.com/imsweb/django-seeker.git into 6.0

Commit:
#1581 - First stab at splitting aggregations out

lefcourn commented on seeker/views.py:
```diff
@@ -415,8 +415,8 @@ def get_view_name(self):
         """
-        Returns a querystring with empty keys removed, keys in sorted order, and values (for keys whose order does not
-        matter) in sorted order. Suitable for saving and comparing searches.
+        Returns a querystring with empty keys removed and keys in sorted order.
```
Are they in sorted order?

lbrant1 replied:
The keys are (from line 426: `for key in sorted(data)`) Although that again seems to serve no useful purpose. What do you think about removing that part as well?

Commit:
#1581 - Finish up splitting aggregations from results search

NOTE: THIS IS A BREAKING CHANGE. Another commit should be coming to repair that before merge.

ruttenb committed:
#1581 - Modified changes so they aren't breaking

ruttenb committed:
#1581 - Removed additional breaking changes

lefcourn committed:
Merge pull request #85 from imsweb/1581_cache_aggregations

#1581 - First stab at splitting aggregations out

ruttenb committed:
Added a search_timeout setting to allows seeker searches to increase
allowed search time before a timeout occurs

lefcourn committed:
Merge pull request #86 from imsweb/search_timeout

Added a search_timeout setting to allows seeker searches to increase

smithlc committed:
increment seeker version

smithlc committed:
Changed version to 4.4.0

smithlc committed:
Update views.py

fix typo

ruttenb committed:
Merge pull request #83 from smithlc/Add_field_definitions

Add field definitions

ruttenb committed:
Created a util function to help convert a SavedSearch querystring to a
search_object.  Added AdvancedSavedSearch to django admin

ruttenb committed:
Bumped seeker version to 4.4.1

tomlinsj committed:
Merge pull request #88 from imsweb/saved_search_conversion_function

Created a util function to help convert a SavedSearch querystring to a

Commit:
Merge branch '4.0' of https://github.com/imsweb/django-seeker.git into
6.0

Conflicts:
	seeker/__init__.py
	seeker/views.py

ruttenb committed:
Make sure keywords get applied to aggregation_search
